src/os/bluestore/BlueStore.cc

```diff
@@ -8106,6 +8352,54 @@ void BlueStore::_txc_finalize_kv(TransContext *txc, KeyValueDB::Transaction t)
     fm->release(p.get_start(), p.get_len(), t);
   }
 
+  if (!bdev_fast) {
+    _txc_update_store_statfs(txc);
+    return;
+  }
```

Comment:
goto end of put the below in a block; otherwise we duplicate code

Reply:
It turns out that simply using goto statement doesn't work because of compile errors (cross initialization of variables).
Instead, I surrounded the latter part of _txc_finalize_kv() with curly braces to avoid code duplication.